cpu/efm32/periph/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config EFM32_XTIMER_USE_LETIMER
```

Comment:
Shouldn't this only be shown when using xtimer?

Reply:
You are right!